pkgs/development/python-modules/robotframework-seleniumlibrary/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, python, robotframework, selenium, mockito, robotstatuschecker, approvaltests }:
+
+buildPythonPackage rec {
+  version = "3.2.0";
+  pname = "robotframework-seleniumlibrary";
+
+  # no tests included in PyPI tarball
+  src = fetchFromGitHub {
+    owner = "robotframework";
+    repo = "SeleniumLibrary";
+    rev = "v${version}";
+    sha256 = "106dl0qgf52wqk1xn4ghj7n2fjhaq0fh2wlnqn29aczbv5q581y3";
+  };
+
+  propagatedBuildInputs = [ robotframework selenium ];
+  checkInputs = [ mockito robotstatuschecker approvaltests ];
```

Comment:
Would testing robotframework-seleniumlibrary with `approvaltests` start an interactive diff/merge tool (whether graphical or CLI), when the respective tests fail?

Reply:
I think it will depend whether if the approval reporter is available (e.g. non-sandboxed builds), otherwise it will just throw an `ApprovalException` exception